docs/DISPLAY_EXPERIENCE.md

```diff
@@ -33,3 +33,69 @@ try {
 
 >**NOTE:** The launchEcosystem function is not a one-time initialization function, you must call it each time you want to display a feature in KinEcosystem side.
 
+
+#### Launch Backup And Restore Flows ###
+Optionally, your app can launch the Kin Backup or Restore flows and receive callbacks when user either completed successfully or failure.
+
+* Step 1 - Create `backupAndRestore` to handle responses:
+Call `Kin.getBackupAndRestoreManager(activity)`
+```java
+    try {
+        backupAndRestore = Kin.getBackupAndRestoreManager(MainActivity.this);
+    } catch (ClientException e) {
+        // sdk not initialized
+    }
+```
+
+* Step 2 - Add callbacks for backup or restore flow as you wish:
+Call `backupAndRestore.registerBackupCallback(callback)` or `backupAndRestore.registerRestoreCallback`
```

Comment:
need to mention that registerCallback need to happen on `onCreate` for the cases of activity restart

Reply:
👍